AuthoringToolTest/PresentationLogic/ModalDialog/ModalDialogInputFieldsFactoryUt.cs

```diff
@@ -9,6 +15,277 @@ namespace AuthoringToolTest.PresentationLogic.ModalDialog;
 
 public class ModalDialogInputFieldsFactoryUt
 {
+    [Test]
+    public void GetCreateLearningElementInputFields_Test()
+    {
+        var systemUnderTest = new ModalDialogInputFieldsFactory();
+
+        for (int contentFileTypeTest = 0; contentFileTypeTest < 8; contentFileTypeTest++)
+        {
+            string name = "foo";
+            string contentFileExtension =
+                helper_GetCreateLearningElementInputFields_ReturnsExpectedFileExtension(contentFileTypeTest);
+
+            var contentType =
+                helper_GetCreateLearningElementInputFields_ReturnsExpectedContentType(contentFileTypeTest);
```

Comment:
Statt dieser Methoden einfach die richtige Endung/ContentType beim TestCase Attribut in die Methode mitgeben